pyfar/classes/coordinates.py

```diff
@@ -273,19 +273,25 @@ def get_cart(self, convention='right', unit='met', convert=False):
                 y, z, x = sph2cart(pts[..., 0], pts[..., 1], pts[..., 2])
 
             else:
+                # Can not be tested. Will only be raised if a coordinate system
```

Comment:
If it can not be tested, what would be a situation, when this error is raised? Maybe leave it out?

Reply:
This would be raised if we add a new coordinate system later, but forget to implement the conversion for it. I would thus leave it in.